linters/eslintrc

```diff
@@ -0,0 +1,44 @@
+env: 
+  browser: true
+  node: true
+  amd: true
+  jquery: true
```

Comment:
I reckon these things trigger errors, when you are trying to do `var $ = require('jquery')` (fails because you are trying to re-define global `$`). So I usually keep all 3rd party libs as `false` even when I actually have them.